app/oauth.go

```diff
@@ -354,7 +354,7 @@ func (a *App) newSession(appName string, user *model.User) (*model.Session, *mod
 
 func (a *App) newSessionUpdateToken(appName string, accessData *model.AccessData, user *model.User) (*model.AccessResponse, *model.AppError) {
 	// Remove the previous session
-	<-a.Srv.Store.Session().Remove(accessData.Token)
+	a.Srv.Store.Session().Remove(accessData.Token)
```

Comment:
I know the return value has been ignore in the old code. But since you are touching this: would you please check the error and log it?